library/amazoncorretto

```diff
@@ -2,6 +2,11 @@ Maintainers: Amazon Corretto Team <corretto-team@amazon.com> (@corretto),
              Dan Lutker <lutkerd@amazon.com> (@lutkerd),
              Ben Taylor <benty@amazon.com> (@benty-amzn),
              Joshua Cao <joshcao@amazon.com> (@caojoshua)
+             David Alvarez <alvdavi@amazon.com> (@alvdavi)
+             Rui Li <ruiamzn@amazon.com> (@rgithubli)
+             Sergey Bylokhov <bylokhov@amazon.com> (@mrserb)
+             Tianmin Shi <tianshi@amazon.com> (@TianminShi)
+             Victor Rudometov <vicrud@amazon.com> (@Rudometov)
```

Comment:
Sorry, this is missing some commas :sweat_smile:

```suggestion
             Joshua Cao <joshcao@amazon.com> (@caojoshua),
             David Alvarez <alvdavi@amazon.com> (@alvdavi),
             Rui Li <ruiamzn@amazon.com> (@rgithubli),
             Sergey Bylokhov <bylokhov@amazon.com> (@mrserb),
             Tianmin Shi <tianshi@amazon.com> (@TianminShi),
             Victor Rudometov <vicrud@amazon.com> (@Rudometov)
```

(It's technically a comma-separated field where whitespace is ignored.)

Reply:
Good catch. Amended in the second commit https://github.com/docker-library/official-images/pull/13183/commits/b292ee3010496f7783f9cde0debcfb1bdbc781d1